### PR TITLE
Minor backporting script tweaks

### PR DESCRIPTION
**Pull request by twpayne**
Just a couple of things to make life easier for backporters :)

### CODE REVIEW

File: contrib/backporting/common.sh [hunk 1]
```diff
@@ -27,3 +27,10 @@ get_remote () {
   fi
   echo "$remote"
 }
+
+require_linux() {
+  if [ "$(uname)" != "Linux" ]; then
+      echo "$0: Linux required"
+      exit 1
+  fi
+}
```
Comment (pchaigno) [inline contrib/backporting/common.sh]: Nit: Looks like a missing newline here.

Do you know what limits those scripts to Linux only?

**Review by christarazi — APPROVED** [review]

File: contrib/backporting/common.sh (hunk 1 above)
Comment (twpayne) [reply contrib/backporting/common.sh]: Newline added.

I haven't investigated why the scripts only run on Linux, but I did observe that they didn't run on my macOS machine. The [backporting process documentation](https://docs.cilium.io/en/latest/contributing/release/backports/#one-time-setup) states:

> The scripts referred to below need to be run on Linux, they do not work on macOS.

**Review by pchaigno — APPROVED** [review]

**Review by tklauser — APPROVED** [review]

**Review by fristonio — APPROVED** [review]
LGTM

**Comment by gandro** [comment]
Marking this for backport, because `submit-backport` is executed from the backport branches.

Note to backporters: You only need to backport commit 05eabc71a8525fffba8977eabdfd6e763dcd08c7  - `tools/dev-doctor` is not available on older branches.

**Pull request merged**